experiment/autobumper/bumper/bumper.go

```diff
@@ -38,26 +42,197 @@ const (
 	testImagePrefix = "gcr.io/k8s-testimages/"
 	prowRepo        = "https://github.com/kubernetes/test-infra"
 	testImageRepo   = prowRepo
+
+	latestVersion          = "latest"
+	upstreamVersion        = "upstream"
+	upstreamStagingVersion = "upstream-staging"
+	tagVersion             = "vYYYYMMDD-deadbeef"
+
+	upstreamBranch           = "https://raw.githubusercontent.com/kubernetes/test-infra/master"
```

Comment:
Nit, this is not a branch name, rename to something like downloadingurlbase

Reply:
Changed to `upstreamURLBase`, which I feel is a better name.